.github/workflows/build.yml

```diff
@@ -0,0 +1,85 @@
+name: CI
+
+# Controls when the workflow will run
+on:
+  # Triggers the workflow on push or pull request events but only for the master branch
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  Windows:
+    runs-on: windows-latest
+
+    # Steps represent a sequence of tasks that will be executed as part of the job
+    steps:
+      # Checks-out your repository under $GITHUB_WORKSPACE, so your job can access it
+      - uses: actions/checkout@v2
+
+      - name: Get Submodules
+        run: git submodule update --init --recursive
+
+      - name: Build
+        run: ./ci-build.cmd
+
+      - name: Upload x64 Release
+        uses: actions/upload-artifact@v2
+        with:
+          name: win-x64
+          path: cimgui\build\x64\Release\cimgui.dll
+
+      - name: Upload x86 Release
+        uses: actions/upload-artifact@v2
+        with:
+          name: win-x86
+          path: cimgui\build\x86\Release\cimgui.dll
+
+      - name: Upload Json Files
+        uses: actions/upload-artifact@v2
+        with:
+          name: JsonFiles
+          path: cimgui\generator\output\*.json
```

Comment:
This is to make sure everything is in one spot rather than opening cimgui URL to grab these files.